tools/rna_tools/htseq-clip/htseq_clip.xml

```diff
@@ -0,0 +1,263 @@
+                #end if
+                #if $action_type.hce_options.filter_bed:
+                    --filter-bed '$action_type.hce_options.filter_bed'
+                    --filter-mode $action_type.hce_options.filter_mode
+                #end if
+                $action_type.hcc_options.hcc_unstranded
+                --no-zipper
+        #end if
+
+    ]]></command>
+
+    <inputs>
+        <conditional name="action_type">
+        
+            <param name="action_type_selector" type="select" label="Select an action">
+                <option value="create_sliding_windows" selected="true">Create sliding windows</option>
+                <option value="create_count_table">Create count table </option>
+            </param>
+
+            <when value="create_sliding_windows">
+
+                <param name="gff_file" type="data" format="gff3"
+                       label="GFF annotation file"
+                       help="Provide a genomic annotation file in GFF3 format"/>
+                <param name="gff_unsorted" label="Is the GFF file unsorted?" type="boolean"
+                       truevalue="--hca-unsorted" falsevalue="" checked="False"
+                       help="Check if GFF file is unsorted (default: GFF file is assumed to be sorted)"/>
+
+                <section name="hcw_options" title="Sliding window settings">
+                    <param name="hcw_w" type="integer" value="50"
+                           label="Sliding window size"
+                           help="Set the sliding window size in nucleotides. If unsure, try 75-100 (default: 50)"/>
+                    <param name="hcw_s" type="integer" value="20"
+                           label="Sliding window step size"
+                           help="Set the sliding window step size (default: 20)"/>
+                </section>
+
+                <section name="win_out_options" title="Output options">
+                    <param name="annot_bed_out" label="Output annotation BED file" type="boolean"
+                               checked="False"
+                               help="Output annotation BED file used for creating sliding windows"/>
+                </section>
+
+            </when>
+
+            <when value="create_count_table">
+                <repeat name="exp_bams" min="1" title="CLIP-seq experiment BAM inputs">
+                    <param name="exp_bam" type="data" format="bam" label="BAM files belonging to the CLIP-seq experiment" help="Select BAM file belonging to the CLIP-seq experiment. NOTE that order determines replicate numbering in output tables"/>
+                </repeat>
+                <repeat name="ctr_bams" min="1" title="CLIP-seq control BAM inputs">
+                    <param name="ctr_bam" type="data" format="bam" label="BAM files belonging to the CLIP-seq control" help="Select BAM file belonging to the CLIP-seq control. NOTE that order determines replicate numbering in output tables"/>
```

Comment:
You can simply use multiple=true and avoid the repeat

Reply:
Wolfgang suggested this in the tech tips chat, as I want to keep the order of the input files (rep1, rep2 ... ).